callback_plugins/ansible_version_check.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding:utf-8 -*-
+# https://adamj.eu/tech/2016/08/19/restricting-the-ansible-version-in-use/
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+
+import ansible
+
+try:
+    # Version 2.0+
+    from ansible.plugins.callback import CallbackBase
+except ImportError:
+    CallbackBase = object
+
+
+def print_red_bold(text):
```

Comment:
from `constants.py` it defines `COLOR_ERROR` with a default value of the string `red`. I think you could get away with calling `_display.display(...., color='red')`. The difference being that we are overriding any custom user configs (which we were doing anyway using the shell escape codes :) )